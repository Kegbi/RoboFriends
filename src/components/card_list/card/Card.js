import React from "react";
import './card.scss';

function Card({name, id, email}) {
  return(
    <div className={'card'}>
      <img src={`https://robohash.org/${id}?set=set3`} alt="kitty"/>
      <h2 className={'card__header'}>{name}</h2>
      <p className={'card_mail'}>{email}</p>
    </div>
  )
}

export default Card;