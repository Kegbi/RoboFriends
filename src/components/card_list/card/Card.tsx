import React from "react";
import "./card.scss";

interface CardStatelessProps {
	name: string;
	email: string;
	id: number;
}

const Card: React.SFC<CardStatelessProps> = ({ name, id, email }) => {
	return (
		<div className={"card"}>
			<img src={`https://robohash.org/${id}?set=set3`} alt="kitty" />
			<h2 className={"card__header"}>{name}</h2>
			<p className={"card_mail"}>{email}</p>
		</div>
	);
};

export default Card;
