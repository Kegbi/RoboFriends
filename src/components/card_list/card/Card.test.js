import React from "react";
import Card from "./Card";
import renderer from 'react-test-renderer';

it('Expect to render Card component', () => {
  const tree = renderer.create(<Card/>).toJSON()
  expect(tree).toMatchSnapshot();
})