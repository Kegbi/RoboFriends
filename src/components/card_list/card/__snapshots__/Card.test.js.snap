// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Card component 1`] = `
<div
  className="card"
>
  <img
    alt="kitty"
    src="https://robohash.org/undefined?set=set3"
  />
  <h2
    className="card__header"
  />
  <p
    className="card_mail"
  />
</div>
`;
